src/olympia/files/helpers.py

```diff
@@ -33,6 +33,21 @@
 LOCKED = 'locked'
 LOCKED_TIMEOUT = 60 * 5
 
+SYNTAX_HIGHLIGHTER_ALIAS_MAPPING = {
+    'xul': 'xml',
+    'rdf': 'xml',
+    'jsm': 'js',
+    'json': 'js',
```

Comment:
I know this was in the old code as well, but do we really need it? Iiuc https://github.com/syntaxhighlighter/brush-javascript/blob/master/brush.js#L43 indicates that `json` does not need to be mapped.

Or are we even using the built-in JavaScript brush ourselves? https://github.com/mozilla/addons-server/blob/master/static/js/zamboni/files.js#L230-L254 defines our own JS brush, extending from the built-in one. Maybe we have to add `json` as an alias there?